docs/peer-testing.md

```diff
@@ -0,0 +1,170 @@
+# [Datadog .NET AppSec] Peer-testing
+
+This document describes how to test the installation of the .NET App Sec Library
+
+## Testing Locally
+
+This assumes the agent is already running, it requires agent 7.31.0-rc.6 or later.
+
+Install .NET (yes, it does run on Mac and Linux): https://dotnet.microsoft.com/download
+
+Create the app
+
+```
+dotnet new webapp -n DotnetPeerTest
+```
```

Comment:
Rather than having these instructions, can we create the sample app directly? That way we can setup all the env vars and everything automatically too (like we do in this sample for example https://github.com/DataDog/dd-trace-dotnet/tree/master/tracer/samples/AutomaticTraceIdInjection/MicrosoftExtensionsExample)

Reply:
The idea is to have someone outside of the .NET group test the installation and basic functionality, we want to simulate the steps that a customer would go through. I therefore think have a prebuilt app would make that too easy.

Reply:
Yeah, makes sense. I think having a sample _as well_ makes for better documentation 🙂